### ISSUE TITLE
Remove unused affixes

### ISSUE THREAD

**Issue by fonsleenaars** [fonsleenaars/tqdb#7]
There seem to be affixes in the parser that aren't actually available for any equipment, these should be removed.

**Comment by fonsleenaars** [comment]
The parser is actually the one responsible for excluding affixes based on whether or not they are used, the website will automatically reflect this when they're no longer in the data set.

**Comment by Quanyails** [comment]
I'm not sure if they should be removed from the database entirely. Maybe they can be made to appear in custom games. Completing #6 might implicitly complete this issue as well; if an affix has no equipment type associated with it, it can be seen to be unused. (I'm not actually sure if that's how it works in the in-game database.)

**Comment by fonsleenaars** [comment]
Indeed, #6 should end up supplying a list of unused prefixes. One option is to leave them in but mark them as unused.
